tdb/crick_upload.py

```diff
@@ -43,15 +43,18 @@ def convert_xls_to_csv(path, fstem, ind):
     workbook = xlrd.open_workbook(path+fstem + exts[ind])
     for sheet in workbook.sheets():
         with open('data/tmp/{}_{}.csv'.format(fstem, sheet.name), 'wb') as f:
-            try:
-                writer = csv.writer(f)
-                print(sheet.name)
-                writer.writerows( sheet.row_values(row) for row in range(sheet.nrows))
-            except:
-                print("couldn't write data/tmp/{}_{}.csv".format(fstem,sheet.name))
+            writer = csv.writer(f)
+            print(sheet.name)
+            for row in range(sheet.nrows):
+                new_row = []
+                for cell in sheet.row_values(row):
+                    try:
+                        new_row.append(unicode(cell).encode('utf-8'))
+                    except:
+                        import pdb; pdb.set_trace()
```

Comment:
@barneypotter24 You left in the try/except and debugging tracing from testing. It should probably go away.

Reply:
It would also be worth including a comment above the use of `unicode(cell).encode('utf-8')` to explain why we need to convert the character/unicode string to a byte/utf-8 string.